packages/jumpstarter-driver-tasmota/jumpstarter_driver_tasmota/driver_test.py

```
@@ -0,0 +1,29 @@
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)